utils/wrappers.py

```diff
@@ -30,20 +28,20 @@ def specific_init(self):
         """initializes necessary attributes depending on the environment"""
 
         if self.env_name == "MountainCar-v0":
-            self.num_actions = self.env.action_space.n # 2 possible actions
+            self.num_actions = self.env.action_space.n
         elif self.env_name == "Hopper-v2":
             self.num_actions = self.env.action_space.shape[0]
         elif self.env_name == "VideoPinballNoFrameskip-v4":
-            self.env = WarpFrame(self.env)
-            self.num_actions = self.env.action_space.n # 9 possible actions
+            self.env = atari_wrapper(self.env_name)
```

Comment:
The `RobustRewardEnv` for VidPin is primarily used by Rainbow. I think Rainbow maybe already added some wrappers, so all that was needed on top was a WarpFrame. I'd suggest using a training pipeline like Rainbow to debug if you want to make this change.

Reply:
Makes sense. I changed the wrap frame to more wrappers because I wanted to have exactly the same setup as for your pytorch trainings. To fix this I would suggest:
1. including the latest version of rainbow and PPO to the codebase, like in an `optimizer.py` file (Ryan)
2. moving the other pytorchy wrappers from `wrappers.py` to `models.py` (Michaël)

Reply:
Rainbow (in the `ptan` repo) uses many different files and has extra dependencies. If you want to merge this now then I suggest switching back to WarpFrame.

Reply:
Got it, will switch back to WarpFrame now, we'll see later for Rainbow.